lib/services/sku-pack-service.js

```diff
@@ -371,6 +371,22 @@ function skuPackServiceFactory(
             });
     };
 
+    SkuPackService.prototype.getPackInfo = function(skuid) {
+        var self = this;
+        return self.loader.getAll(self.confRoot)
+        .then(function (conf) {
+            var skuConf = conf[skuid + '.json'];
+            if(skuConf) {
+                var json = JSON.parse(skuConf);
+                return {
+                    description: json.description || null,
+                    version: json.version || null
+                };
+            }
+            return null;
```

Comment:
Because there is no sku pack.  You want me to reflect that no pack is installed by pretending one is installed without a description or version defined?